tests/engines/collections/acceptance/submit/submit-test.ts

```diff
@@ -0,0 +1,64 @@
+import { click } from '@ember/test-helpers';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { percySnapshot } from 'ember-percy';
+import { module, test } from 'qunit';
+
+import { Permission } from 'ember-osf-web/models/osf-model';
+import { visit } from 'ember-osf-web/tests/helpers';
+import { setupEngineApplicationTest } from 'ember-osf-web/tests/helpers/engines';
+
+module('Collections | Acceptance | submit', hooks => {
+    setupEngineApplicationTest(hooks, 'collections');
+    setupMirage(hooks);
+
+    test('it renders', async assert => {
+        server.loadFixtures('taxonomies');
+        server.loadFixtures('licenses');
+        const currentUser = server.create('user', 'loggedIn');
+        const primaryCollection = server.create('collection');
+        const nodeToBeAdded = server.create('node', {
+            title: 'Node to be added to collection',
+            currentUserPermissions: Object.values(Permission),
+        });
+        server.create('contributor', {
+            node: nodeToBeAdded,
+            users: currentUser,
+            index: 0,
+        });
+        const taxonomies = server.schema.taxonomies.all().models;
+        const licensesAcceptable = server.schema.licenses.all().models;
+        const provider = server.create('collection-provider', {
+            id: 'studyswap',
+            primaryCollection,
+            taxonomies,
+            licensesAcceptable,
+        });
+        await visit(`/collections/${provider.id}/submit`);
+        await percySnapshot(assert);
+        await click('.ember-power-select-placeholder');
+        await click('.ember-power-select-option');
+        await percySnapshot(assert);
```

Comment:
When you have multiple snapshots in one test, you have to give each a unique name, otherwise it just keeps the first snapshot and discards all the others (see [this build](https://percy.io/Center-for-Open-Science/ember-osf-web/builds/2180481)). You do this by passing in a string for the name to `percySnapshot()` instead of `assert`, e.g.
```ts
await percySnapshot('Collections | Acceptance | submit | Project metadata');
```
If you don't want to keep repeating the module name, you can do something like I did in the meetings acceptance tests:
https://github.com/CenterForOpenScience/ember-osf-web/blob/119c41f898f74a32600d8f6cd97dd7eb50e41057/tests/acceptance/meetings/index-test.ts#L1-L26